src/dataservice/data-service-dao-file/src/main/java/com/redhat/depdraw/dataservice/dao/file/FileUtil.java

```diff
@@ -0,0 +1,31 @@
+package com.redhat.depdraw.dataservice.dao.file;
+
+import java.io.File;
+
+public class FileUtil {
+    public static final String BASE_FILES_DIR = "~/opt/DepDraw/data/";
```

Comment:
what about having configurable properties in application.properteis, at least for the root folder?
this way we can have a %prod% value for OpenShift deployment and a %dev% value for local testing